### ISSUE TITLE
[BUG] MAIN: Brand logo(s)

### ISSUE THREAD

**Issue by dudlyart** [Redlion-news/issues#7]
**Describe the bug**
We have set both width and height on the logo, I re-upload with shadow, and it's deformed now. We may remove the height parameter (if it's ok) and make it little bigger (i tried 15rem). See image

**Environnement**
- [X] Production
- [ ] Staging

![image](https://user-images.githubusercontent.com/81701877/198036644-ba113cf6-2a28-44d7-9ccd-64339ad3f2ca.png)

**Comment by qruz-hq** [comment]
Yup, sure thing, very subtle but does change a lot to the eye. What a bull's eye, this amazing graphic design has.hehe
